repo-projects/Directory.Build.targets

```
@@ -447,8 +447,23 @@
     </ItemGroup>
   </Target>
 
+  <UsingTask TaskName="Microsoft.DotNet.UnifiedBuild.Tasks.ReadRepoInfoFromSourceManifest" AssemblyFile="$(MicrosoftDotNetUnifiedBuildTasksAssembly)" TaskFactory="TaskHostFactory" />
+  <Target Name="SetRepoOriginalSourceRevisionId">
+    <ReadRepoInfoFromSourceManifest SourceManifest="$(SrcDir)/source-manifest.json"
```

nit: SrcDir already has a trailing directory separator char

```suggestion
    <ReadRepoInfoFromSourceManifest SourceManifest="$(SrcDir)source-manifest.json"
```